examples/power_spectra_example.py

```diff
@@ -0,0 +1,43 @@
+"""
+===============================
+POWER SPECTRUM OF A TIMESERIES
+===============================
+
+An example showing how to estimate power spectrum of a TimeSeries
+"""
+
+##############################################################################
+# Start by importing the necessary modules.
+
+import matplotlib.pyplot as plt
+import pandas as pd 
+from scipy import signal
+
+import sunpy.timeseries
+from sunpy.data.sample import GOES_XRS_TIMESERIES
+
+###############################################################################
+# Let's first load a TimeSeries from sample data
+
+ts = sunpy.timeseries.TimeSeries(GOES_XRS_TIMESERIES)
```

Comment:
You might want to describe the data you will working on to give whoever reads this an idea of what the input is.

Reply:
I have added a brief description about the input data now  . Is it okay ?